job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContextSpec.groovy

```diff
@@ -3065,25 +3072,57 @@ class PublisherContextSpec extends Specification {
         exception.message =~ /\(.+, line \d+\) jobIdentifier cannot be null or empty/
 
         where:
-        id   | _
```

Comment:
Data tables are a nice syntactic sugar for data pipes, but it doesn't look very readable for just a single variables.